node/examples/electron/semantic_segmentation/forge.config.js

```diff
@@ -0,0 +1,40 @@
+const fse = require('fs-extra');
+
+module.exports = {
+  packagerConfig: {
+    prune: true,
+    asar: {
+      unpack: "node_setup.js",
+      unpackDir: "{lib,build}"
+   },
+  },
+  makers: [
+    {
+      name: '@electron-forge/maker-zip'
+    }
+   ],
+  hooks: {
+    generateAssets: async () => {
+      // Copy WebNN libs
+      await fse.copy('../../../node_setup.js', 'node_setup.js', {overwrite: true});
```

Comment:
Define a path variable of node like `let node_path = '../../../';`

Reply:
Thanks, comment addressed.